config.json

```diff
@@ -559,7 +559,13 @@
       "unlocked_by": null,
       "difficulty": 1,
       "topics": [
-
+        "logic",
+        "parsing",
+        "pattern matching",
```

Comment:
We've switched to `snake_case` naming convention for topics to increase consistency among tracks, so could you please choose them from the following list - https://github.com/exercism/problem-specifications/blob/master/TOPICS.txt

Reply:
fixed. I followed the convention used in the json-file, seems like there are mixed naming conventions in there.

Reply:
@miickel yeah, it's inconsistent, that's why we're encouraging people the list I provided above